hooks/cache_location.py

```diff
@@ -13,11 +13,18 @@
 """
 
 import sgtk
-from sgtk import TankError
 import os
-import errno
 
 from tank_vendor.shotgun_base import get_pipeline_config_cache_root
+from tank_vendor.shotgun_base.utils import (
```

Comment:
this is a hook, so we should make sure to use as "public" methods as possible. If we import directly from `tank_vendor.shotgun_base` and `tank_vendor.shotgun_deploy` rather than from the internals of those libs, we are less likely to break things in the future. 

It's not explicitly spelled out in the API yet (and we need docs!) - sorry! But i was sort of thinking that you would only every import `shotgun_base` and `shotgun_deploy` - whatever is on that level is the public interface - does that make sense?